### PR TITLE
add existing_property_value and passing test

### PR DESCRIPTION
**Pull request by ros3bud**
Solves #17, #19, #20, #21

@cgrayson or @alexkwolfe, LGTY? 

- pass lead.income to monthly_salary
- pass lead.occ_stat to employment_status
- pass lead.bal_two to mortgage.second_mortgage_balance
- pass lead.est_val to mortgage.existing_property_value
- add request.variables
- add passing tests

### CODE REVIEW

**Comment by cgrayson** [comment]
Oh, is this ready to review? Or are you still committing to it?

**Comment by ros3bud** [comment]
@cgrayson -  I will let you know when ready.

File: src/inbound.coffee [hunk 1]
```diff
@@ -161,13 +172,30 @@ selectMimeType = (contentType) ->
...
+  switch value 
+    when 'employed', 'self_emp', 'retired', 'unemployed' then value
+    else null
+
+parseLatePayments = (value) ->
+  return unless value?
+  value = value.toLowerCase()
+  switch value 
+    when 'none', 'one', 'one_to_two', 'two_or_more', 'three_or_more' then value
+    else null
```
Comment (cgrayson) [inline src/inbound.coffee]: The LC field [mortgage.application.late_payments](mortgage.application.late_payments) is a number.

**Comment by cgrayson** [comment]
Otherwise LGTM

**Pull request merged**